### ISSUE TITLE
Add ESLint

### ISSUE THREAD

**Issue by vinitshahdeo** [vinitshahdeo/MiniYouTube#1]
> **Linting is the process of checking the source code for Programmatic as well as Stylistic errors.** 

Adding ESLint will be helpful in identifying some common and uncommon mistakes that are made during coding. 

> **ESLint covers both code quality and coding style issues.** 

- **Add ESLint to this project.**
- Add `scripts` in `package.json` i.e. `npm run test-lint` should run lint tests.
- Do not make any changes to other files.

#### Please refer [this](https://eslint.org/).

**Comment by grigori-gru** [comment]
I'll do this

**Comment by kalpananegi** [comment]
is this issue open?

**Comment by vinitshahdeo** [comment]
@kalpananegi Yes, give it a shot or you may look into a few **[good first issues](https://vinitshahdeo.github.io/HacktoberFest2K19/)** which are **[up for grabs](https://github.com/vinitshahdeo/HacktoberFest2K19).**

### :white_check_mark: [Link to the repository](https://github.com/vinitshahdeo/HacktoberFest2K19)

**Comment by kushal-khare-official** [comment]
is this issue still open? I would like to do this.

**Comment by arielhad** [comment]
@vinitshahdeo I have a branch ready but I need permission to create a branch and open the PR

**Comment by mvpowers** [comment]
@aleoof @arielhad You have to create a fork of this repository, make the commits on your fork, push them up, and then create a pull request.

You will not have permission issues anymore because you are the maintainer of the fork.

Here is an article with more information: https://opensource.com/article/19/7/create-pull-request-github

**Comment by arielhad** [comment]
@vinitshahdeo @mvpowers  Can you please review my PR:
 https://github.com/vinitshahdeo/MiniYouTube/pull/27

**Comment by Akshit42-hue** [comment]
Can I work upon this issue under hacktoberfest @vinitshahdeo

**Comment by tanwi1234** [comment]
I have added the ESLint .please review my PR @vinitshahdeo
